test/e2e/specs/editor/blocks/spacer.spec.js

```diff
@@ -48,4 +48,21 @@ test.describe( 'Spacer', () => {
 			)
 		).toBeFocused();
 	} );
+
+	test( 'should work in theme without spacing units support', async ( {
+		admin,
+		editor,
+		requestUtils,
+	} ) => {
+		await requestUtils.activateTheme( 'twentytwenty' );
```

Comment:
Switching themes can be expensive, usually done in `test.beforeAll` hooks. Can we find a different way to test this without switching themes?